src/ProgramBuilder.ts

```diff
@@ -186,11 +194,15 @@ export class ProgramBuilder {
             if (event === 'add' || event === 'change') {
                 const fileObj = {
                     src: thePath,
-                    dest: rokuDeploy.getDestPath(thePath, this.program.options.files, this.rootDir)
+                    dest: rokuDeploy.getDestPath(
+                        thePath,
+                        this.program.options.files,
+                        //some shells will toTowerCase the drive letter, so do it to rootDir for consistency
+                        util.driveLetterToLower(this.rootDir)
```

Comment:
This is the core of this PR. The other stuff was just a few small tweaks to make the CLi a little more friendly.